File: solutions/csharp/armstrong-numbers/ArmstrongNumbers.cs
public static class ArmstrongNumbers
{
    public static bool IsArmstrongNumber(int number)
    {
        string digits = number.ToString();
        return digits
            .Select(x => x.ToString())
            .Select(int.Parse)
            .Sum(x => (int)Math.Pow(x, digits.Length))
            .Equals(number);
    }
}

File: solutions/javascript/armstrong-numbers/armstrong-numbers.js
export const isArmstrongNumber = (number) => {
  const bigNumber = BigInt(number)
  const digits = bigNumber.toString().split('').map(Number)
  return bigNumber == digits.reduce(
    (acc, digit) => acc + BigInt(digit) ** BigInt(digits.length),
    BigInt(0)
  );
};
